WebRoot/js/zimbraMail/prefs/view/ZmZimletsPage.js

```diff
@@ -555,11 +555,17 @@ ZmPrefZimletListView.prototype._getCellId = function(item, field, params) {
 	return DwtId.getListViewItemId(DwtId.WIDGET_ITEM_CELL, "zimlets", item.name, field);
 };
 
+ZmPrefZimletListView.prototype._getCellClass = function (item, field) {
+	if (field === ZmPrefZimletListView.COL_ACTIVE) {
+		return "ZmAlignCenter";
```

Comment:
As discussed, the thought was here for having a generic class name so that other column items can align their content in the middle. And, after sharing ideas ideas after discussion, 
changed the name to ZmPrefCheckboxColumn.
https://github.com/Zimbra/zm-web-client/pull/98/commits/6a6a14dd2f05f961c3d5d8d962c046d40c19d080